website/markdown/docs/contribution/scratch.mdx

```diff
@@ -1,13 +1,16 @@
 ---
 name: Scratch
-order: 11
 excerpt: Random ideas and notes for Tuist. Pseudocode and braindumps.
 ---
 
 # Scratch
 
+## March 31th 2020 - Compiling ProjectDescription with the Swift version of the minimum supported version of Xcode
```

Comment:
I added this page to the documentation to do brain dumps and keep track of the decisions that we are making while evolving and growing the project.